docs/tokencredentials.md

```diff
@@ -48,6 +48,10 @@ string[] scopes = {"User.Read"};
 DeviceCodeCredentialOptions deviceCodeCredentialOptions = new DeviceCodeCredentialOptions()
 {
     ClientId = clientId,
+    DeviceCodeCallback = (info, cts) => {
```

Comment:
interesting, last time I checked the java version didn't have a default (might have been fixed since) so I added it mechanically. I think it's nice to demonstrate to people they can display things anywhere they want without needing them to do some research.